translations/ja.po

```diff
@@ -432,7 +432,7 @@ msgstr ""
 
 #: src\commands\rush-duel.ts:114
 msgid "**MAXIMUM ATK**: ${ card.maximum_atk }"
-msgstr ""
+msgstr "**最大攻撃力**: ${ card.maximum_atk }"
```

Comment:
Uh, I'm not sure this is how this mechanic is described in the source material.

https://yu-gi-oh.jp/news_detail.php?page=details&id=928

Reply:
Now I see what this means. Happy to know more about rush duel XD